File: rest/usage-triggers/instance-get-example-1/instance-get-example-1.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'rubygems'          # This line not needed for ruby > 1.8
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = 'ACed732a3c49700934481addd5ce1659f0'
auth_token = '{{ auth_token }}'
@client = Twilio::REST::Client.new account_sid, auth_token

# Get an object from its sid. If you do not have a sid,
# check out the list resource examples on this page
@trigger = @client.account.usage.triggers.get("UT33c6aeeba34e48f38d6899ea5b765ad4")
puts @trigger.current_value

File: rest/notification/instance-get-example-1/instance-get-example-1.js
// Download the Node helper library from twilio.com/docs/node/install
// These vars are your accountSid and authToken from twilio.com/user/account
var accountSid = '{{ account_sid }}';
var authToken = "{{ auth_token }}";
var client = require('twilio')(accountSid, authToken);

client.notifications("NO5a7a84730f529f0a76b3e30c01315d1a").get(function(err, notification) {
    console.log(notification.messageText);
});